app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -418,7 +343,23 @@ function toField(name: string, valueType?: ValueType | null | undefined): ColDef
     case 'Mixed':
       icon = 'mixed'
   }
-  const svgTemplate = `<svg viewBox="0 0 16 16" width="16" height="16"> <use xlink:href="${icons}#${icon}"/> </svg>`
+
+  const dataQuality =
+    typeof props.data === 'object' && 'data_quality_pairs' in props.data ?
+      props.data.data_quality_pairs
+    : { number_of_nothing: [], number_of_whitespace: [] }
+  const nothingIsZeroOrNull =
+    index != null ? !(dataQuality.number_of_nothing && dataQuality.number_of_nothing[index]) : true
+  const whitespaceIsZeroOrNull =
+    index != null ?
+      !(dataQuality.number_of_whitespace && dataQuality.number_of_whitespace[index])
+    : true
+
+  const hideDataQuality = nothingIsZeroOrNull && whitespaceIsZeroOrNull
```

Comment:
I like to stick to the JS convention that "value absent" is falsy and "value present" is truthy--in this case that would mean inverting the logic of this and its sub-computations, like:
`const showDataQuality = nothingIsNonZero || whitespaceIsNonZero`